pandas/_libs/src/ujson/python/JSONtoObj.c

```diff
@@ -35,14 +35,12 @@ Numeric decoder derived from from TCL library
  * Copyright (c) 1994 Sun Microsystems, Inc.
 */
 
-// "py_defines.h" needs to be included first to
-// avoid compilation errors, but it does violate
-// styleguide checks with regards to include order.
-#include "py_defines.h"
 #define PY_ARRAY_UNIQUE_SYMBOL UJSON_NUMPY
 #define NO_IMPORT_ARRAY
-#include <numpy/arrayobject.h>  // NOLINT(build/include_order)
-#include <ultrajson.h>          // NOLINT(build/include_order)
+#define PY_SSIZE_T_CLEAN
```

Comment:
This definition is suggested by the Python developers:

https://docs.python.org/3/c-api/intro.html#include-files

I noticed we haven't used anywhere else so could probably be added as a follow up in other places